controllers/rules/rules.go

```diff
@@ -74,6 +74,21 @@ const ConfigurationRules = `
             cpu: LARGEST_VALUE
             memory: LARGEST_VALUE
             ephemeral-storage: LARGEST_VALUE
+- name: common-service-postgresql
+  resources:
+  - apiVersion: postgresql.k8s.enterprisedb.io/v1
+    kind: Cluster
+    name: common-service-db
+    data:
+      spec:
+        instances: LARGEST_VALUE
+        resources:
+          limits:
+            cpu: LARGEST_VALUE
+            memory: LARGEST_VALUE
+          requests:
+            cpu: LARGEST_VALUE
+            memory: LARGEST_VALUE    
```

Comment:
My apologies on missing this....
Could we add a `ephemeral-storage: LARGEST_VALUE` here as well?